tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Exporter.cs

```diff
@@ -25,6 +25,13 @@ internal static partial class ConfigurationKeys
         /// <seealso cref="ExporterSettings.AgentUri"/>
         public const string AgentPort = "DD_TRACE_AGENT_PORT";
 
+        /// <summary>
+        /// Configuration key for setting the timeout in milliseconds for sending traces to the Agent.
+        /// Default value is <c>15,000</c>.
+        /// </summary>
+        /// <seealso cref="ExporterSettings.TracesTimeoutMs"/>
+        public const string TracesTimeoutMs = "DD_TRACE_TIMEOUT_MS";
```

Comment:
I was just looking to see what other tracers use for this, we currently have:

- Java `DD_TRACE_AGENT_TIMEOUT` seconds, (default 10)
- PHP `DD_TRACE_AGENT_CONNECT_TIMEOUT` and `DD_TRACE_AGENT_TIMEOUT` milliseconds (default 100 and 500 respectively) 

So maybe we should use `DD_TRACE_AGENT_TIMEOUT` too? Though take your pick on s vs ms 🙄

Reply:
Good eye on the other tracers, time to make one more change 😅 Since Java is the big one, I'll default to Java's name and implementation. Sorry for the repeated changes but this should be the last! And thank you for looking into the other tracers 🙇🏼